.github/workflows/examples.yml

```diff
@@ -32,6 +32,9 @@ jobs:
 
         # Install all dependencies needed for examples.
         pip install --progress-bar off $(ls dist/*.tar.gz)[example]
+
+        # Install additional test-specific dependencies
+        pip install --progress-bar off nbval
```

Comment:
Is it okay to add test-specific dependencies here, or shall I move to examples' requirements?

Reply:
I think that it's not a bad idea to install `nbval` here, but for future maintainability, it seems better to add it to the "example" dependency list defined in [setup.py](https://github.com/optuna/optuna/blob/master/setup.py#L57).